lib/sass_spec/test.rb

```diff
@@ -51,6 +53,8 @@ def handle_expected_error_message!(test_case, options)
   expected_error_msg = _extract_error_message(test_case.expected_error, options)
 
   if expected_error_msg != error_msg
+    return false if options[:probe_todo] && test_case.todo?
+    return false if options[:probe_todo] && test_case.warning_todo?
```

Comment:
It returns false, so it indicates a failure IMO. It just does not trigger the assertion, so that flag can be stored to `result?`. In the end I want the same behavior as without `--run-todo`, but to report which todo tests would actually have passed.